controllers/admin.js

```diff
@@ -0,0 +1,69 @@
+const AdminModel = require('../models/admin.js')
+const UserModel = require('../models/user.js')
+// const UserModel = require('../models/user.js')
+const dotenv = require ('dotenv')
```

Comment:
Remove require("dotenv") from every file except main file that is executed for the first time.

Reply:
removed